### ISSUE TITLE
FromObject Failing in .NET 5

### ISSUE THREAD

**Issue by GravlLift** [lukemurray/EntityGraphQL#70]
I'm not real clear on the specifics, but when including this package as a dependency on a ASP.NET project running dotnet 5.0.0-rc.1, calls to SchemaBuilder.FromObject will result in a `MissingMethodException`:
```
System.MissingMethodException
  HResult=0x80131513
  Message=Method not found: 'System.String Humanizer.InflectorExtensions.Singularize(System.String, Boolean)'.
  Source=EntityGraphQL
  StackTrace:
   at EntityGraphQL.Schema.SchemaBuilder.AddFieldWithIdArgumentIfExists[TContextType](SchemaProvider`1 schema, Type contextType, Field fieldProp) in EntityGraphQL\src\EntityGraphQL\Schema\SchemaBuilder.cs:line 116
   at EntityGraphQL.Schema.SchemaBuilder.FromObject[TContextType](SchemaProvider`1 schema, Boolean autoCreateIdArguments, Boolean autoCreateEnumTypes, Func`2 fieldNamer) in EntityGraphQL\src\EntityGraphQL\Schema\SchemaBuilder.cs:line 70
```
If I clone the repo and upgrade the Humanizer.Core library from the current 2.7.9 to version 2.8.2 or higher, the code runs without issue. Seems a simple fix.

**Comment by AceDevelopmentSystems** [comment]
I am getting the same  

```
System.MissingMethodException: Method not found: 'System.String Humanizer.InflectorExtensions.Singularize(System.String, Boolean)'.
   at EntityGraphQL.Schema.SchemaBuilder.AddFieldWithIdArgumentIfExists[TContextType](SchemaProvider`1 schema, Type contextType, Field fieldProp)
   at EntityGraphQL.Schema.SchemaBuilder.FromObject[TContextType](SchemaProvider`1 schema, Boolean autoCreateIdArguments, Boolean autoCreateEnumTypes, Func`2 fieldNamer)
   at EntityGraphQL.Schema.SchemaBuilder.FromObject[TContextType](Boolean autoCreateIdArguments, Boolean autoCreateEnumTypes, Func`2 fieldNamer)
   at Ace.Aceterilize.Startup.ConfigureServices(IServiceCollection services) in C:\Development\Ace.Aceterilize\Ace.Aceterilize\Startup.cs:line 54
```


My start up  

```
  public void ConfigureServices(IServiceCollection services)
        {

            services.Configure<FeaturesConfiguration>(Configuration.GetSection("Features"));
            services.AddDbContext<AceterilizeDbContext>(options =>
            {
                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"))
                .UseLazyLoadingProxies();
          }).AddDefaultIdentity<ApplicationUser>()
                .AddEntityFrameworkStores<AceterilizeDbContext>();

            services.AddSingleton(SchemaBuilder.FromObject<AceterilizeDbContext>());//Line 54
```
Please let me know if you need further info

**Comment by lukemurray** [comment]
Thanks for the info. I haven't played with .net 5 yet, but now it's not in beta I will have a look.

**Comment by lukemurray** [comment]
Fixed by #80
